vaxrank/core_logic.py

```diff
@@ -211,26 +160,29 @@ def vaccine_peptides_for_variant(
     At this point, we know the variant has RNA support, as per the
     isovar_protein_sequence.
     """
-    protein_fragment = MutantProteinFragment.from_isovar_protein_sequence(
-        variant=variant,
-        protein_sequence=isovar_protein_sequence)
+    if not isovar_result.passes_all_filters:
+        # don't consider candidate vaccine peptides from variants which either
+        # failed their filters or don't have an RNA-derived protein sequence
+        return []
+
+    variant = isovar_result.variant
+    long_protein_fragment = MutantProteinFragment.from_isovar_result(isovar_result)
```

Comment:
why rename this to "long"? is the meaning different from before?

Reply:
I wanted to distinguish the 35mer source sequence from the 25mer vaccine peptides below